Orange/util.py

```diff
@@ -33,11 +33,10 @@ def deprecated(obj):
 
     Examples
     --------
-    >>> import sys; old_stderr, sys.stderr = sys.stderr, sys.stdout  # So doctest can catch that warning
     >>> @deprecated
     ... def old():
     ...     return 'old behavior'
-    >>> old()
+    >>> old()  # doctest: +SKIP
     /... OrangeDeprecationWarning: Call to deprecated ... old ...
     'old behavior'
```

Comment:
'C:\\... OrangeDeprecationWarning' which is the problem (doctest ELLIPSIS cannot match at the beginning of the output line)